app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 30
```

Comment:
I can't do this until scoped storage is added to support external directories.

Reply:
Can put it to 29 first but you should always try to use the latest ask version to get the latest improvements and to be allowed to publish to play

Reply:
The current version is the last one without issues if I remember correctly. I don't publish to Play so that doesn't matter, but if I didn't care about the latest improvements I wouldn't be migrating from ButterKnife. I just hate dealing with API changes from Google so I haven't touched scoped storage yet.

Reply:
Scoped storage should be super easy so maybe I'll do a follow-up PR

Reply:
I also don't know how you do the download, but depending on that you don't even need to change anything

Reply:
Right now I only use it for the image cache, but at some point songs and other data will probably go there.

Reply:
Well that's something to implement then, right? For image caching it shouldn't be a problem

Reply:
Well, I don't need to implement anything :p unless I feel like wasting my free time adding scoped storage. I wanted the caching outside so it would persist across installs, although I'm not actually sure if that happens.

Reply:
Well at some time you will have to do it 🤷‍♂️ except you want to hang on apiLevel 29 for eternity
Also if you don't cache in the chance folder the clear cache function doesn't work correctly (which is pretty bad)

Reply:
Also I think there was some discussion about enforcing it for all apps so that might be coming too

Reply:
It actually can't even be 29 it has to be 28 for things to work.

Reply:
Fixed